tests/platform_tests/mellanox/mellanox_thermal_control_test_helper.py

```diff
@@ -913,6 +923,12 @@ def mock_data(self):
         platform_data = get_platform_data(self.mock_helper.dut)
         thermal_dict = platform_data["thermals"]
         for category, content in thermal_dict.items():
+            # TODO: temp local solution, since the "sodimm" is only supported in master,
```

Comment:
why not checking on branch instead of todo?

Reply:
This is also the original code which I did not modified. It was removed for some reason when I am handling the updates according to pre-commit check result in the seconds commit. I readd it again.